src/Sql/dbo/Stored Procedures/Cipher_SoftDelete.sql

```diff
@@ -24,11 +24,13 @@ BEGIN
         AND [Id] IN (SELECT * FROM @Ids)
 
     -- Delete ciphers
+    DECLARE @UtcNow DATETIME2(7);
+    SET @UtcNow = GETUTCDATE();
```

Comment:
I think you can declare and set in one line. Should be how it is done elsewhere in sprocs.

```suggestion
    DECLARE @UtcNow DATETIME2(7) = GETUTCDATE();
```